util/log.go

```diff
@@ -64,6 +70,12 @@ func GetCheckerEntries(prefix string, logger *log.Logger) []ContextEntry {
 	}
 }
 
+// SentryWarnf implements LogFunc, notifying sentry of an error.
+func SentryWarnf(ctx context.Context, format string, v ...interface{}) {
```

Comment:
we shouldbe using sentry for errors only, no need to send warnings that we ignore most of the time anyway. Let's get the KV errors as error